src/executor/Scheduler.ts

```diff
@@ -23,21 +23,24 @@ export class Scheduler {
         'tsE2E': 'ts',
         'less': 'less',
         'css': 'css',
-        'openAPIYaml': 'yaml'
+        'openAPIYaml': 'yaml',
+        'vendor': 'package-lock.json|index.ts'
```

Comment:
In this case we can't watch the file extension only (e.g. json), because watch would be then triggered when we don't want. For example by changing the `libraryLicenseInfos.json`.

picomatch should use simple regex [test](https://github.com/micromatch/picomatch#test) method. Similar to [this](https://jsfiddle.net/z8pehg0v/2/)